File: TCDNew-master/src/ui.wince/Cen.Wms.Client/Models/State/SettingsFacility.cs
using Cen.Wms.Client.Models.Enums;

namespace Cen.Wms.Client.Models.State
{
    class SettingsFacility
    {
        public string FacilityId { get; set; }
        public AcceptanceProcessType AcceptanceProcessType { get; set; }
        public string PalletCodePrefix { get; set; }
        public bool IsAcceptanceByPapersEnabled { get; set; }
    }
}

File: TCDNew-master/src/ui.android/lib/models/state/SettingsFacility.dart
import 'package:tiq/models/enums/AcceptanceProcessType.dart';

class SettingsFacility {
  final String facilityId;
  final AcceptanceProcessType acceptanceProcessType;
  final String palletCodePrefix;
  final bool isAcceptanceByPapersEnabled;

  SettingsFacility(this.facilityId, this.acceptanceProcessType, this.palletCodePrefix, this.isAcceptanceByPapersEnabled);
}